sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InheritedCollidingInterfaceMembers.cs

```diff
@@ -84,4 +84,10 @@ public interface IMultilayer : IReadOnlyDictionary<string, object>, IDummy // Co
     public interface IDummy
     {
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/3225
+    // Issue is raised correctly, but message does not contain name, because indexers doesn't have an identifier
```

Comment:
I would remove this comment (// Issue is raised correctly, but message does not contain name, because indexers doesn't have an identifier) it is no longer valid and does not help.